README.md

```diff
@@ -1,2 +1,19 @@
-# vllm-spyre
-vLLM Spyre Plugin
+# Spyre plugin for vLLM
+
+This plugin enable support for IBM Spyre accelerator in [vLLM](https://docs.vllm.ai/en/latest/).
+
+## Installation from source with Docker
+
+First, download vllm-spyre
+
+```
+git clone https://github.com/IBM/vllm-spyre
+cd vllm-spyre
+```
+
+Build image from source
+
+```
+docker build . -f Dockerfile.spyre -t vllm-spyre
+docker run -it --rm vllm-spyre bash
```

Comment:
I think we should also explain how to install the plugin into a local environment, rather than just how to build the docker image.

Reply:
Done